social-network/src/articles-view.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import YourArticle from './articles-yours';
+import MissedArticle from './articles-missed';
+import articleDataYours from './your-articles.json';
+import articleDataMissed from './missed-articles.json';
+import PropTypes from 'prop-types';
+
+export default class ViewArticles extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render () {
+        const articleYours = articleDataYours.map((article, index) =>     
+                    <YourArticle key={index} article={article} />
+        );
+
+        const articleMissed = articleDataMissed.map((article, index) => 
+                    <MissedArticle key={index} article={article} />    
+        );
+
+        return (
+            <div>
+                <h2 className="header">For you</h2>
+                        <span className="grid-yours">
+                            {articleYours}
+                        </span>
+                <h2 className="header">In case you missed it</h2>
+                        <span className="grid-missed">
+                            {articleMissed}
+                        </span>
+            </div>
+        )
+        
+    }
+}
+
+YourArticle.propTypes = {
+    articleYours: PropTypes.object,
+    articleMissed: PropTypes.object

```

Comment:
The syntax here is wrong. You are telling react that there is a component <YourArticle/> that expects 2 props to be passed. articleYours and articleMissed. 
`YourArticle.propTypes = {
    article: PropTypes.object
}
MissedArticle.propTypes = {
    article: PropTypes.object
}`

Reply:
Where you first target the component and then name all the props it expects to receive and what type of data should be passed for each.